packages/a11y/src/test/index.test.ts

```diff
@@ -96,10 +100,12 @@ describe( 'speak', () => {
 
 		afterEach( () => {
 			setup();
-			containerPolite = document.getElementById( 'a11y-speak-polite' );
+			containerPolite = document.getElementById(
+				'a11y-speak-polite'
+			) as HTMLElement;
```

Comment:
Apart from this, we will need to use optional chainning inside `beforeEach` above.

```suggestion
			);
```